modules/Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -409,6 +466,102 @@ private void captureYield(List<MovecraftLocation> harvestedBlocks) {
             }
         }
     }
+    
+    private boolean processNetherPortal(Block block) {
```

Comment:
The style on this method is all over the place, also use nullability annotations.